vllm/entrypoints/openai/protocol.py

```diff
@@ -211,8 +211,8 @@ class ChatCompletionRequest(OpenAIBaseModel):
     stop: Optional[Union[str, List[str]]] = Field(default_factory=list)
     stream: Optional[bool] = False
     stream_options: Optional[StreamOptions] = None
-    temperature: Optional[float] = 1.0
-    top_p: Optional[float] = 1.0
+    temperature: Optional[float] = None
+    top_p: Optional[float] = None
```

Comment:
I understand why you want these to be None by default, but I think it is most clear to have the default value here. I won't block based on this

Reply:
If a default value is set, fastapi will use the default value diretly, i guess a comment is enough?

Reply:
We can't set a default value here because it depends on generation config.